static/js/common.js

```diff
@@ -0,0 +1,11 @@
+  // Fix bootstrap backdrop stacking when having multiple modals
```

Comment:
might wanna move this logic into the `explorer.js`, which is loaded on all pages too and contains the other common stuff :)